pysmiles/smiles_helper.py

```diff
@@ -876,6 +877,42 @@ def _mark_chiral_atoms(molecule):
             neighbours = [neighbours[0],  neighbours[1], neighbours[3], neighbours[2]]
         molecule.nodes[node]['rs_isomer'] = tuple(neighbours)
 
+def check_for_ez_conflicts(anchor, tagged_nodes, ez_isomer_class):
+    n1, n2 = tagged_nodes
+    if (n1 < anchor and n2 < anchor) or (n1 > anchor and n2 > anchor):
+        assert ez_isomer_class[n1] != ez_isomer_class[n2]
+    else:
+        assert ez_isomer_class[n1] == ez_isomer_class[n2]
+
+def annotate_ez_isomers(molecule, ez_atoms):
```

Comment:
I don't like that it has the same name as `_annotate_ez_isomers`, and this functions should be private as well.

Reply:
It's a terrible name but I did not have it in me yesterday evening to give it a more sensible name. Perhaps `interpret_cis_trans_tokens` for the second one?

Reply:
Works for me!